addons/mrp/report/mrp_report_bom_structure.py

```diff
@@ -27,7 +25,10 @@ def _compute_current_production_capacity(self, bom_data):
         components_qty_to_produce = defaultdict(lambda: 0)
         components_qty_available = {}
         for comp in bom_data.get('components', []):
-            if comp['product'].type != 'product' or float_is_zero(comp['base_bom_line_qty'], precision_digits=comp['uom'].rounding):
+            if comp['product'].type != 'product' or float_is_zero(
+                    comp['base_bom_line_qty'],
+                    precision_rounding=comp['uom'].rounding,
```

Comment:
```suggestion
                comp['base_bom_line_qty'],
                precision_rounding=comp['uom'].rounding,
```

Reply:
Will be removing this change anyway once this PR is approved & merged into master: https://github.com/odoo/odoo/pull/163154